plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableDataStoreTests.php

```
@@ -2277,6 +2277,234 @@ public function test_create_update_date_gmt() {
+		$order = new WC_Order();
+		$this->switch_data_store( $order, $this->sut );
+		$order->set_created_via( 'checkout' );
+		$order->save();
+
+		// Get the original creation date from the order object.
+		$original_date_created_obj = $order->get_date_created();
+		$this->assertNotNull( $original_date_created_obj, 'Order should have a creation date.' );
+		$original_date_created = $original_date_created_obj->setTimezone( new DateTimeZone( 'UTC' ) )->format( 'Y-m-d H:i:s' );
+
+		$orders_table = OrdersTableDataStore::get_orders_table_name();
+		$order_id = $order->get_id();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$order_id'. (undefined)

(UnusedLocalVariable)